lib/uncruft/deprecation_handler.rb

```diff
@@ -44,6 +44,10 @@ def normalize_callstack_path(message)
         message.gsub!(gem_home, '$GEM_PATH')
       end
 
+      if (ruby_home = ruby_home(message)).present?
+        message.gsub!(ruby_home, bin_path(ruby_home))
+      end
```

Comment:
```suggestion
      if message.include? RbConfig::CONFIG['bindir']
        message.gsub!(RbConfig::CONFIG['bindir'], '$BIN_PATH')
      end
```

I don't know if that will work as intended, but I'm curious if it will -- I suspect that matching on the `bindir` config could be a lot simpler than what we do for `gem_home`, because the gem path matching has some extra folders in between gem home and where the gems get installed (and we found that they were varying depending on the system, which meant we needed a more complicated regex match and replace strategy).